### PR TITLE
PWGGA/GammaConv: GammaIsoTree added Rho subtraction & lim. acc.

### PR DESCRIPTION
**Pull request by fjonasALICE**
- added rho subtraction and limited acceptance for charged isolation

### CODE REVIEW

File: PWGGA/GammaConv/AliAnalysisTaskGammaIsoTree.cxx [hunk 1]
```diff
@@ -2244,10 +2261,18 @@ void AliAnalysisTaskGammaIsoTree::UserExec(Option_t *){
   fDataEvtHeader.pVtxY = vertex[1];
   fDataEvtHeader.pVtxZ = vertex[2];
   if(outrho){ // ugly workaround until problem is fixed
-    fDataEvtHeader.rho = outrho->GetVal();
+    fChargedRho = outrho->GetVal();
```
Comment (fjonasALICE) [inline PWGGA/GammaConv/AliAnalysisTaskGammaIsoTree.cxx:2264]: important part

File: PWGGA/GammaConv/AliAnalysisTaskGammaIsoTree.cxx [hunk 2]
```diff
@@ -2366,7 +2391,8 @@ void AliAnalysisTaskGammaIsoTree::ProcessConversionPhotons(){
 
 
     if(!fConvCuts->PhotonIsSelected(PhotonCandidate,fInputEvent)) continue;
-    
+    Bool_t isWithinTPC =IsWithinRadiusTPC(PhotonCandidate->Eta(),PhotonCandidate->Phi(),fExclusionRadius);
```
Comment (fjonasALICE) [inline PWGGA/GammaConv/AliAnalysisTaskGammaIsoTree.cxx:2394]: important part

File: PWGGA/GammaConv/AliAnalysisTaskGammaIsoTree.cxx [hunk 3]
```diff
@@ -3124,12 +3150,17 @@ void AliAnalysisTaskGammaIsoTree::ProcessCaloPhotons(){
...
+     // Subtract correction of charged isolation
+     // others not implemented yes
+  
 
      // when writing to tree, only fill first two R
      for (UInt_t r = 0; r < isoCharged.size(); r++)
      {
-       if(r<2) tmp_isoCharged[r] = isoCharged.at(r);
+       // Fill charged isolation corrected for background
+       if(r<2) tmp_isoCharged[r] = isoCharged.at(r) - fChargedRhoTimesArea[r];
```
Comment (fjonasALICE) [inline PWGGA/GammaConv/AliAnalysisTaskGammaIsoTree.cxx:3163]: important part

**Comment by nschmidtALICE** [comment]
+1

**Pull request merged**